src/amo/components/Home/index.js

```diff
@@ -0,0 +1,185 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { setViewContext } from 'amo/actions/viewContext';
+import Link from 'amo/components/Link';
+import {
+  CLIENT_APP_ANDROID,
+  CLIENT_APP_FIREFOX,
+  VIEW_CONTEXT_HOME,
+} from 'core/constants';
+import translate from 'core/i18n/translate';
+import Card from 'ui/components/Card';
+
+import './styles.scss';
+
+
+export const CategoryLink = ({ children, name, slug, type }) => {
+  return (
+    <li className={classNames('Home-category-li', `Home-${name}`)}>
+      <Link to={`/${type}/${slug}/`} className="Home-category-link">
+        <span>{children}</span>
+      </Link>
+    </li>
+  );
+};
+CategoryLink.propTypes = {
+  children: PropTypes.node.isRequired,
+  name: PropTypes.string.isRequired,
+  slug: PropTypes.string.isRequired,
+  type: PropTypes.string.isRequired,
+};
+
+export const ExtensionLink = (props) => {
+  return <CategoryLink type="extensions" {...props} />;
+};
+
+export const ThemeLink = (props) => {
+  return <CategoryLink type="themes" {...props} />;
+};
+
+export class HomeBase extends React.Component {
+  static propTypes = {
+    clientApp: PropTypes.string.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    i18n: PropTypes.object.isRequired,
+  }
+
+  componentWillMount() {
+    const { dispatch } = this.props;
+
+    dispatch(setViewContext(VIEW_CONTEXT_HOME));
+  }
+
+  extensionsCategoriesForClientApp() {
+    const { clientApp, i18n } = this.props;
+
+    let linkHTML = null;
+
+    if (clientApp === CLIENT_APP_ANDROID) {
+      linkHTML = (
+        <ul className="Home-category-list">
+          <ExtensionLink name="block-ads" slug="security-privacy">
+            {i18n.gettext('Block ads')}
+          </ExtensionLink>
+          <ExtensionLink name="screenshot" slug="photos-media">
+            {i18n.gettext('Screenshot')}
+          </ExtensionLink>
+          <ExtensionLink name="find-news" slug="feeds-news-blogging">
+            {i18n.gettext('Find news')}
+          </ExtensionLink>
+          <ExtensionLink name="shop-online" slug="shopping">
+            {i18n.gettext('Shop online')}
+          </ExtensionLink>
+          <ExtensionLink name="be-social" slug="social-networking">
+            {i18n.gettext('Be social')}
+          </ExtensionLink>
+          <ExtensionLink name="play-games" slug="sports-games">
+            {i18n.gettext('Play games')}
+          </ExtensionLink>
+        </ul>
+      );
+    }
+
+    if (clientApp === CLIENT_APP_FIREFOX) {
```

Comment:
Yeah, the thing is the category API just returns everything... I guess we could request filters on it though.